### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines.

### ISSUE THREAD

**Issue by CherylEvans** [department-of-veterans-affairs/va.gov-team#64437]

**Comment by CherylEvans** [comment]
This ticket was recreated from: department-of-veterans-affairs/va.gov-cms#[11364](https://app.zenhub.com/workspaces/platform-governance-team-5f85b91c14d8df0018fac414/issues/gh/department-of-veterans-affairs/va.gov-cms/11364)
508-defect-3 [COGNITION]: Top 3 buttons should be styled as links.

**Comment by laflannery** [comment]
Completed in [#14928](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14928)
